### DISCUSSION TITLE
What about previous custom prefix / suffix?

### DISCUSSION THREAD

**Discussion by antoniogiroz** [dm4t2/vue-currency-input#270] [Q&A] [answered]
Hello,

Thanks for creating this component, it's awesome!

I have been studying this component for a few days (previous to release 2) and in addition to using it for currencies, I was also want to use it for percentages with the custom suffix.

But now, the currency is mandatory (that's why it's called vue-currency-input haha) ... and the prefix and suffix options are no longer available, Do you have any workaround to use this component for percentages and not have to make me a component of my own?

Thanks!

**Comment by dm4t2** [comment, marked as answer]
Hi, you already guessed it. This change is intended, see also #177 and #209. 

I recently started to work on a more generic number input based on the ECMAScript Internationalization API, which will also support raw decimal formats with custom prefixes/suffixes, units, and even percentages:

https://github.com/dm4t2/intl-number-input

It is in a very early state and still needs a lot of work.

**Reply by ragulka** [reply]
This seems like exactly the thing I'm looking for. I have been using https://github.com/autoNumeric/autoNumeric for numeric inputs until now, but it has it's own issues and does not work with Vue 3 at all.

I was about to set out to build my own implementation for an app, but as I was digging through both of your codebases (this one here and https://github.com/dm4t2/intl-number-input), I realized there's a lot of edge case stuff that needs to be handled - so I'd rather avoid building everything from scratch myself.

I'd be happy to help out with https://github.com/dm4t2/intl-number-input in any way, if needed/possible.
